man/SpaDES.shiny-package.Rd

```diff
@@ -11,7 +11,7 @@ The \pkg{SpaDES.shiny} package
 \seealso{
 Useful links:
 \itemize{
-  \item \url{https://predictiveecology.github.io/SpaDES.shiny/}
+  \item \url{http://spades.shiny.predictiveecology.org/}
```

Comment:
Didn't we agree that we won't be applying this change (which is automatically done by Roxygen)? This new page is under development. The deleted one is operating.

Reply:
Fixed

Reply:
the correct URL is http://spades-shiny.predictiveecology.org